tests/unit/core/components/TestInstallButton.js

```diff
@@ -82,146 +105,113 @@ describe(__filename, () => {
+  it('renders a Button for extensions', () => {
+    const installURL = 'https://a.m.o/files/addon.xpi';
+    const addon = createInternalAddonWithInstallURL({ installURL });
 
-    const history = addQueryParamsToHistory({
-      history: createMemoryHistory(),
-    });
+    const root = render({ addon });
 
-    return mount(
-      <I18nProvider i18n={i18n}>
-        <Router history={history}>
-          <InstallButton {...props} />
-        </Router>
-      </I18nProvider>,
-    );
-  };
-
-  it('renders InstallSwitch when mozAddonManager is available', () => {
-    const installURL = 'https://a.m.o/files/addon.xpi';
-    // This can't use createFakeAddon({ files: [...] }) because it needs
-    // to specify a custom object for addon.current_version.
-    const addon = createInternalAddon({
-      ...fakeTheme,
-      current_version: {
-        ...fakeTheme.current_version,
-        files: [
-          {
-            ...fakeTheme.current_version.files[0],
-            platform: OS_ALL,
-            url: installURL,
-          },
-        ],
-      },
-    });
-    const root = render({
-      hasAddonManager: true,
-      addon,
-      // Simulate how disco/components/Addon spreads addon theme data.
-      ...addon,
-    });
     expect(root.type()).toEqual('div');
     expect(root).toHaveClassName('InstallButton');
-    expect(root).toHaveClassName('InstallButton--use-switch');
 
-    const switchComponent = root.childAt(0);
-    expect(switchComponent.type()).toEqual(InstallSwitch);
+    const button = root.find(Button);
 
-    expect(switchComponent).toHaveClassName('InstallButton-switch');
-    expect(switchComponent).toHaveProp('addon', addon);
-    expect(switchComponent).toHaveProp('installURL', installURL);
+    expect(button).toHaveLength(1);
+    expect(button.childAt(1)).toHaveText('Add to Firefox');
+    expect(button).toHaveClassName('InstallButton-button');
+    expect(button).not.toHaveClassName('InstallButton-button--uninstall');
+    expect(button).toHaveProp('buttonType', 'action');
+    expect(button).toHaveProp('className', 'InstallButton-button');
+    expect(button).toHaveProp('disabled', false);
+    expect(button).not.toHaveProp('data-browsertheme');
+    expect(button).toHaveProp('href', installURL);
+    expect(button).toHaveProp('onClick', root.instance().installExtension);
+    expect(button).toHaveProp('prependClientApp', false);
+    expect(button).toHaveProp('prependLang', false);
 
-    // Make sure it passes all theme properties.
-    expect(switchComponent.props()).toMatchObject(addon.themeData);
+    const icon = button.find(Icon);
+    expect(icon).toHaveLength(1);
+    expect(icon).toHaveProp('name', 'plus');
   });
 
-  it('renders a theme button when mozAddonManager is not available', () => {
-    const addon = createInternalAddon(fakeTheme);
-    const root = render({ hasAddonManager: false, addon });
+  it('renders a button for themes', () => {
```

Comment:
nit: above you have renders a 'Button' for themes..

also, I am wondering if we should have one for static theme too as this is only specific to lightweight theme I believe